test/unistd/test_unistd_write_broken_link.c

```diff
@@ -10,21 +11,23 @@ int main() {
   printf("link result: %d\n", res);
   int src_fd = open("link_source", O_CREAT | O_WRONLY, 0777);
   printf("source_fd: %d, errno: %d %s\n", src_fd, errno, strerror(errno));
-  write(src_fd, "abc", 3);
+  write(src_fd, "abc\0", 4);
```

Comment:
Seems a little odd to write null byte to a text file.. but I guess its not important here.